db/seeds.rb

```diff
@@ -5,3 +5,15 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+3.times do |num|
```

Comment:
WOW! Seeds are good! But, you should use factory bot for develop's data and use the factories in seeds.